cloud-aws-cloudformation/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/connector/resource/AwsLoadBalancerLaunchService.java

```diff
@@ -185,8 +189,15 @@ private ListStackResourcesResult updateCloudFormationStack(AuthenticatedContext
 
         String cfTemplate = cloudFormationTemplateBuilder.build(modelContext);
         LOGGER.debug("CloudFormationTemplate: {}", cfTemplate);
-        cfClient.updateStack(awsStackRequestHelper.createUpdateStackRequest(ac, stack, cFStackName, cfTemplate));
-
+        try {
+            cfClient.updateStack(awsStackRequestHelper.createUpdateStackRequest(ac, stack, cFStackName, cfTemplate));
+        } catch (AmazonServiceException e) {
+            if (VALIDATION_ERROR.equalsIgnoreCase(e.getErrorCode())) {
+                ValidateTemplateResult result = cfClient.validateTemplate(awsStackRequestHelper.createValidateTemplateRequest(cfTemplate));
```

Comment:
I think it would make more sense to validate the template before the update

Reply:
Only happens few times, so I wouldn't make a validate call every time. This log could uncover the issue and we could implement it later in an earlier validation step.

Reply:
I see, we can keep it that way then. But please add a log for the exception, and I guess we should also re-throw in case it is not a validation error.

Reply:
Fair point! It should be re-thrown anyway. 
It is logged by the AmazonServiceExceptionMapper later.